WEB-INF/classes/gov/noaa/pfel/erddap/util/EDStatic.java

```diff
@@ -6763,4 +6763,92 @@ else if (errorNo == HttpServletResponse.SC_SERVICE_UNAVAILABLE) //http error 503
+        EDStatic.tally.remove("Log in failed (since last daily report)");
+        EDStatic.tally.remove("Log in succeeded (since last daily report)");
+        EDStatic.tally.remove("Log out (since last daily report)");
+        EDStatic.tally.remove("Main Resources List (since last daily report)");
+        EDStatic.tally.remove("Metadata requests (since last daily report)");
+        EDStatic.tally.remove("OpenSearch For (since last daily report)");
+        EDStatic.tally.remove("OutOfMemory (Array Size), IP Address (since last daily report)");
+        EDStatic.tally.remove("OutOfMemory (Too Big), IP Address (since last daily report)");
+        EDStatic.tally.remove("OutOfMemory (Way Too Big), IP Address (since last daily report)");
+        EDStatic.tally.remove("POST (since last daily report)");
+        EDStatic.tally.remove("Protocol (since last daily report)");
+        EDStatic.tally.remove("Requester Is Logged In (since last daily report)");
+        EDStatic.tally.remove("Request refused: not authorized (since last daily report)");
+        EDStatic.tally.remove("Requester's IP Address (Allowed) (since last daily report)");
+        EDStatic.tally.remove("Requester's IP Address (Blacklisted) (since last daily report)");
+        EDStatic.tally.remove("Requester's IP Address (Failed) (since last daily report)");
+        EDStatic.tally.remove("Requester's IP Address (Too Many Requests) (since last daily report)");
+        EDStatic.tally.remove("RequestReloadASAP (since last daily report)");
+        EDStatic.tally.remove("Response Failed    Time (since last daily report)");
+        EDStatic.tally.remove("Response Succeeded Time (since last daily report)");
+        EDStatic.tally.remove("RSS (since last daily report)");
+        EDStatic.tally.remove("Search File Type (since last daily report)");
+        EDStatic.tally.remove("Search For (since last daily report)");
+        EDStatic.tally.remove("SetDatasetFlag (since last daily report)");
+        EDStatic.tally.remove("SetDatasetFlag Failed, IP Address (since last daily report)");
+        EDStatic.tally.remove("SetDatasetFlag Succeeded, IP Address (since last daily report)");
+        EDStatic.tally.remove("SOS index.html (since last daily report)");
+        EDStatic.tally.remove("Subscriptions (since last daily report)");
+        EDStatic.tally.remove("tabledap DatasetID (since last daily report)");
+        EDStatic.tally.remove("tabledap File Type (since last daily report)");
+        EDStatic.tally.remove("WCS index.html (since last daily report)");
+        EDStatic.tally.remove("WMS doWmsGetMap (since last daily report)");
+        EDStatic.tally.remove("WMS doWmsGetCapabilities (since last daily report)");
+        EDStatic.tally.remove("WMS doWmsDemo (since last daily report)");
+        EDStatic.tally.remove("WMS index.html (since last daily report)");
+    }
+
+    public static void resetTimeDistributions() {
```

Comment:
resetDailyDistributions

This doesn't reset all of the distributions on EDStatic